internal/config/config.go

```diff
@@ -60,14 +61,20 @@ func loadConfig() *Config {
 	viper.SetDefault(AllowEmailToDomains, []string{})
 	viper.BindEnv(AllowEmailToDomains, "ALLOW_EMAIL_TO_DOMAINS")
 
+	// Split the string on commas, viper doesn't support doing this to env vars
+	domains := []string{}
+	if viper.GetString(AllowEmailToDomains) != "" {
+		domains = strings.Split(viper.GetString(AllowEmailToDomains), ",")
```

Comment:
would we need to trim the domains?  many people type coma separated list with a space in between

Reply:
Since it's an env var you would have to quote it to even put a space into it, and if you use the config via `zero-aws-eks-stack` it creates the contents of this env var automatically using TF and helm. I'll trim it too though, might as well.

Reply:
oh right 👍 , didnt see the usage from the infra PR at the time
yeah so will never really have that problem in our setup